worldengine/cli/main.py

```diff
@@ -490,7 +490,8 @@ def main():
     elif operation == 'export':
         world = load_world(args.FILE)
         print_world_info(world)
-        export(world, args.export_type, args.export_bpp, args.export_signed, args.export_normalize)
+        print("Dependency missing: imex/export.py")
+        #export(world, args.export_type, args.export_bpp, args.export_signed, args.export_normalize)
```

Comment:
The file imex/export.py is not part of the master-branch. The import crashed the program.